src/components/filterableContactList/filterableContactTable.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ToolBar from '../toolBar/toolBar';
+import ContactList from '../contactList/contactList';
+import contacts from '../../data';
+
+export default class FilterableContactTable extends React.PureComponent {
```

Comment:
Unnecessary component, you can move everything from here to app

Reply:
Done